terraform/ggroup-to-splunk/variables.tf

```diff
@@ -0,0 +1,43 @@
+variable "lambda_zip_location" {
+  description = "Location of the lambda zip"
+  type = string
+  default = ""
+  }
+
+variable "runtime" {
+  description = "runtime for lambda"
+  default     = "python3.7"
+}
+
+variable "region" {
+  type    = string
+  default = "eu-west-2"
+}
+
+variable "lambda_memory" {
+  default = 2048
```

Comment:
That's a lot of memory. Do you *really* need that much?

Reply:
Nah, changed to 256